src/library/Flex/styled.js

```diff
@@ -84,13 +84,13 @@ const flexItemMapValueToProperty = (
 export const FlexItemRoot = styled(Box, {
   shouldForwardProp: (prop) =>
     ['inline', 'minWidth', 'width'].indexOf(prop) === -1
-})(({ alignItems, direction, justifyContent, alignSelf, breakpoints, grow, minWidth, shrink, theme, width }) =>
+})(({ flexBasis, alignItems, direction, justifyContent, alignSelf, breakpoints, grow, minWidth, shrink, theme, width }) =>
   getResponsiveStyles({
     breakpoints,
     mapValueToProperty: flexItemMapValueToProperty,
     styles: {
       alignSelf,
-      flexBasis: width || 'auto',
+      flexBasis: flexBasis || width || 'auto',
```

Comment:
Should we not make it 0 instead of 'auto'?
Or is that too dangerous?
cc @tx-trent

Reply:
@tx-trent  I wonder if this is ultimately the solution to the issue you had as well (with the screen layout)

Reply:
I'm not sure this would fix that issue. It wasn't the flexBasis that was messing with the layout.

Reply:
@tx-nestor @tx-trent 
I tested out changing the default to 0 instead of auto. While I didn't notice any adverse change in `tx-print`, there are a few components in `td-core-web-ui` that have strange wrapping/adverse behavior from making this change. 

I don't think we should change it in this pr, but maybe we look into it in its own pr and address any components that need to be modified as well.